server/svix-server/src/v1/endpoints/application.rs

```diff
@@ -90,7 +90,7 @@ async fn list_applications(
     pagination: ValidatedQuery<Pagination<ApplicationId>>,
     permissions: Permissions,
 ) -> Result<Json<ListResponse<ApplicationOut>>> {
-    let limit = pagination.limit;
+    let limit = pagination.capped_limit();
```

Comment:
See my other comments, the `endpoints/*.rs` files probably shouldn't need to change.